test/native-promises.tap.js

```diff
@@ -0,0 +1,619 @@
+if (!global.Promise) return;
```

Comment:
none without flags.  Not sure if we care about flags, but the instrumentation is pretty straight forward, unless they were completely different, I don't see this being an issue.